src/Common/Shared.csproj

```diff
@@ -13,6 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Domain\SSW.Rewards.Domain.csproj" />
```

Comment:
Shared can't have a dependency on domain...but I can't see this referenced anywhere in the code so assuming this is a mistake.

Reply:
Fixed, accidental import.